### ISSUE TITLE
Vocalization of the number of participants

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/meet#872]
## Bug Report

**Problematic behavior**
When navigating with the keyboard, the "show participants" button does have the correct label, but the number of participants is not announced.

**Expected behavior/code**
The number of participants should be announced when navigating with the keyboard.

<img width="339" height="95" alt="Image" src="https://github.com/user-attachments/assets/d91e65de-ba47-47a3-81cc-1976f3ded7bd" />

**Possible Solution**

**Additional context/Screenshots**
Add any other context about the problem here. If applicable, add screenshots to help explain.
